chucks-pick3-web/static/assets/js/bootstrapFormVal.js

```diff
@@ -7,7 +7,7 @@
         var forms = document.getElementsByClassName('needs-validation');
         // Loop over them and prevent submission
         var validation = Array.prototype.filter.call(forms, function (form) {
-            form.addEventListener('submit', function (event) {
+            forms.addEventListener('submit', function (event) {
```

Comment:
Revert this change back to using just `form` instead of `forms`.

Reply:
Completed changes for form.